composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/yaml": "~2.0"
+        "symfony/yaml": "~2.0|3.0.0@dev"
```

Comment:
I suggest removing ``@dev`` here, and adding a build job on Travis changing the minimum stability. This way, most builds will still run against stable versions rather than the 3.0 dev version.

see https://github.com/kriswallsmith/assetic/blob/37bcae173f2ca774ee125ea378690d192596aa29/.travis.yml#L37 for changing the min stability in a job